src/main/java/gregtech/api/net/NetworkHandler.java

```diff
@@ -223,29 +223,9 @@ public static void init() {
                 }
         ));
 
-        registerPacket(7, CPacketPluginSynced.class, new NetworkHandler.PacketCodec<>(
-                (packet, buf) -> {
-                    MetaTileEntityMonitorScreen screen = packet.plugin.getScreen();
-                    buf.writeVarInt(screen.getWorld().provider.getDimension());
-                    buf.writeBlockPos(screen.getPos());
-                    buf.writeVarInt(packet.id);
-                    if(packet.payloadWriter != null) {
-                        packet.payloadWriter.accept(buf);
-                    }
-                },
-                (buf) -> {
-                    int dim = buf.readVarInt();
-                    BlockPos pos = buf.readBlockPos();
-                    TileEntity te = FMLCommonHandler.instance().getMinecraftServerInstance().getWorld(dim).getTileEntity(pos);
-                    if(te instanceof MetaTileEntityHolder && ((MetaTileEntityHolder) te).getMetaTileEntity() instanceof MetaTileEntityMonitorScreen) {
-                        MonitorPluginBaseBehavior pluginBaseBehavior = ((MetaTileEntityMonitorScreen) ((MetaTileEntityHolder) te).getMetaTileEntity()).plugin;
-                        if (pluginBaseBehavior != null) {
-                            return new CPacketPluginSynced(pluginBaseBehavior, buf);
-                        }
-                    }
-                    return new CPacketPluginSynced(null, buf);
-                }
-        ));
+        CPacketPluginSynced.registerPacket(7);
```

Comment:
Maybe move these packet numbers to `GregtechDataCodes`?

Reply:
blueberry said he will rework a general refactoring of NetworkHandler.